File: 10_week_2/reverseArray.py
arr = [1, 2, 3, 4, 5, 6, 7, 8]

def reverseArray(arr):
	j = len(arr)
	for idx in range(0, j/2, 1):
		# do not need temp variable for python
		arr[idx], arr[j-1-idx] = arr[j-1-idx], arr[idx]

reverseArray(arr)
print(arr)

File: 10_week_2/reverseArray.js
var arr = [1, 2, 3, 4, 5, 6, 7, 8];

function reverseArray(arr) {
	var temp;
	for (var i = 0, j=(arr.length-1); i < j/2; i++) {
		temp = arr[i];
		arr[i] = arr[j-i];
		arr[j-i] = temp;
	}
}

reverseArray(arr);
console.log(arr);
